### PR TITLE
Minor enhancements

### PR DESCRIPTION
**Pull request by leopic-sc**
Hi folks

I made some minor adjustments to the code:
- Minor changes to bring it up (I think?) with the latest JavaScript syntax
- Some TLC to the call to action
- Translated, and I use that word very loosely because I used DeepL, the call to actions. Please feel free to suggest something more fitting

Screenshot 2021-04-13 at 17 52 17

### CODE REVIEW

**Review by bruderstein — APPROVED** [review]
@axinja could you check the german?

File: script.js [hunk 1]
```diff
@@ -155,12 +155,12 @@ window.addEventListener("touchstart", function (e) {
 
 window.addEventListener("load", function () {
 
-    var instructions = document.createElement("section");
+    let instructions = document.createElement("section");
     instructions.className = "instructions";
-    instructions.innerHTML = "<p>Click or Touch with one finger to Start</p>"
+    instructions.innerHTML = "<p>Klicken oder tippen Sie zum Starten</p>"
```
Comment (axinja) [inline script.js]: bare with me finding the right words.
I assume we can stick to a not too formal copy, right?
```suggestion
    instructions.innerHTML = "<p>Klicken oder mit einem Finger tippen um zu starten</p>"
```

File: script.js [hunk 2]
```diff
@@ -155,12 +155,12 @@ window.addEventListener("touchstart", function (e) {
...
 
     if ('ontouchstart' in window) {
-      instructions.innerHTML += '<p class="mobile">Tap the screen with two fingers to start playing sound. Tap with three to stop.</p>'
+      instructions.innerHTML += '<p class="mobile">Tippen Sie mit zwei Fingern auf den Bildschirm, um die Tonwiedergabe zu starten. Tippen Sie mit drei, um zu stoppen.</p>'
```
Comment (axinja) [inline script.js]:
```suggestion
      instructions.innerHTML += '<p class="mobile">Tippe mit zwei Fingern auf den Bildschirm, um die Wiedergabe zu starten, mit drei Fingern, um sie zu stoppen.</p>'
```

**Comment by leopic-sc** [comment]
For posterity, these are the URLs where this fork is living:
- Editor https://codesandbox.io/s/choose-your-instrument-q4git
- Page https://q4git.csb.app/

**Pull request merged**